### ISSUE TITLE
Naming recommendations

### ISSUE THREAD

**Issue by Scotchester** [cfpb/cf-tables#5]
For best alignment with [our BEM conventions](https://github.com/cfpb/front-end/blob/54ca2885503632c198c5f964bcf09f8272a08260/css.md#naming-conventions), I suggest a few things:

Rename `table__small_stacked` to `table__stack-on-small` to follow the `-on-small` convention established in cf-layout. Rename `table__small_entry-header` to `table__entry-header-on-small` for the same reason.

It's also worth pointing out, for future reference, that given our BEM scheme, I read the original classes as "A `stacked` element of a `table` block that has the `small` modifier." This is the reason for the ["Avoid creating elements of modifiers" guideline](https://github.com/cfpb/front-end/blob/54ca2885503632c198c5f964bcf09f8272a08260/css.md#avoid-creating-elements-of-modifiers).

I don't think we're very stringent about our Less variable naming scheme (i.e., we don't have a coherent one), but I don't think we want the underscores in the particular variables here. I'd go with simple single hyphens, e.g.,

``` less
@table-head-bg: #f8f8f8;
@table-cell-bg: #ffffff;
@table-cell-bg-alt: #f8f8f8;
@table-scrolling-border: #e3e4e5;
@table-border: #babbbd;
```

Also, let's go with `.sorted-down` and `.sorted-up` (replacing underscores with dashes).

**Comment by KimberlyMunoz** [comment]
@mistergone is this issue resolved?

**Comment by mistergone** [comment]
Fixed in #12, but not yet merged.

**Comment by mistergone** [comment]
Merged in PR #12!
